tests/test_screenshot.py

```diff
@@ -26,18 +26,27 @@ def required_templates():
         },
     }
 
+def screenshot_and_access():
+    return {
+        "screenshot": {
+            "results": SCREENSHOT_DATA,
+        },
+        "access": {
```

Comment:
I think it's fine to add this to `required_templates` instead of parametrizing

Reply:
For some reason that I cannot figure out if I add `access` to `required_templates` all other test cases fail.
The screenshot template is loaded with the correct param but it returns 0 when we want to close or cancel instead.

Reply:
You found a big issue with the way the test harness starts the templates. It has to load templates into the same process because they are on the same bus, and they are on the same object because that's just how the portal system works, and that means that we call AddTemplate on the same object which means `def load` in all the different templates actually operates on the same python object.

I'll try to find a better solution for this, but in the meantime, move access over screenshot, and it should work.

Reply:
Trying to mitigate the strongest offenders here: https://github.com/flatpak/xdg-desktop-portal/pull/1608